packages/shared/lib/services/sync/manager.service.ts

```diff
@@ -493,13 +493,30 @@ export class SyncManagerService {
         }
     }
 
-    private async syncStatus(sync: Sync, environmentId: number, includeJobStatus: boolean, orchestrator: Orchestrator): Promise<ReportedSyncJobStatus> {
+    private async syncStatus({
+        sync,
+        environmentId,
+        providerConfigKey,
+        includeJobStatus,
+        orchestrator
+    }: {
+        sync: Sync;
+        environmentId: number;
+        providerConfigKey: string;
+        includeJobStatus: boolean;
+        orchestrator: Orchestrator;
+    }): Promise<ReportedSyncJobStatus> {
         const isGloballyEnabled = await featureFlags.isEnabled('orchestrator:schedule', 'global', false);
         const isEnvEnabled = await featureFlags.isEnabled('orchestrator:schedule', `${environmentId}`, false);
         const isOrchestrator = isGloballyEnabled || isEnvEnabled;
         if (isOrchestrator) {
             const latestJob = await getLatestSyncJob(sync.id);
             const schedules = await orchestrator.searchSchedules([{ syncId: sync.id, environmentId }]);
+            let frequency = sync.frequency;
+            if (!frequency) {
+                const syncConfig = await getSyncConfigByParams(environmentId, sync.name, providerConfigKey);
+                frequency = syncConfig?.runs || '';
```

Comment:
what's best to return here: an empty string or null?

Reply:
null